src/api/controllers/articleController.js

```diff
@@ -53,6 +52,110 @@ export default class ArticleController {
     }
   }
 
+  /**
+   * publish a drafted article
+   * @static
+   * @param {Object} req the request
+   * @param {Object} res the response to be sent
+   * @memberof ArticleController
+   * @returns {Object} res
+   */
+  static async publish(req, res) {
+    const { slug } = req.params;
+    const article = await Article.update({
+      status: 'published'
+    },
+    {
+      where: {
+        slug,
+        title: {
+          [Sequelize.Op.ne]: null,
+        },
+        body: {
+          [Sequelize.Op.ne]: null,
+        },
+        description: {
+          [Sequelize.Op.ne]: null,
+        },
+        category: {
+          [Sequelize.Op.ne]: null,
+        },
+        coverImage: {
+          [Sequelize.Op.ne]: null,
+        },
+      }
+    });
+    const published = 1;
+    if (article[0] === published) {
+      res.status(OK).json({
+        message: 'congratulations for making it, now you article is published',
+      });
+    } else {
+      errorSender(BAD_REQUEST, res, 'Properties', errorMessage.missingProperty);
+    }
+  }
+
+  /**
+   * This function gets user's drafted articles
+   *
+   * @static
+   * @param {Object} req the request
+   * @param {Object} res the response to be sent
+   * @memberof Articles
+   * @returns {Object} res
+   */
+  static async getDraftedArticles(req, res) {
+    const defaultLimit = 10;
+    const defaultOffset = 0;
+    const result = {};
+    const emptyArticleArray = 0;
+
+    const limit = req.query.limit || defaultLimit;
+    const offset = req.query.offset || defaultOffset;
+    const articleAttributes = [
```

Comment:
why do you need to declare these attributes if you know they are going to be fetched exactly as you declared them @alainburindi ?

Reply:
I didn't see it bro